sdk/src/main/scala/com/horizen/account/state/WithdrawalMsgProcessor.scala

```diff
@@ -13,14 +12,20 @@ import scorex.util.serialization.{Reader, Writer}
 import scala.collection.JavaConverters.seqAsJavaListConverter
 import scala.util.{Failure, Success, Try}
 
-object WithdrawalMsgProcessor extends AbstractFakeSmartContractMsgProcessor {
+trait WithdrawalRequestProvider extends AbstractFakeSmartContractMsgProcessor {
```

Comment:
I think WithdrawalRequestProvider should NOT extend AbstractFakeSmartContractMsgProcessor
Instead `object WithdrawalMsgProcessor extends WithdrawalRequestProvider with WithdrawalRequestProvider`

Reply:
Done